kuma/users/jinja2/users/ban_user_and_cleanup_summary.html

```diff
@@ -0,0 +1,87 @@
+{% extends "base.html" %}
+
+{% block body_attributes %}{% endblock %}
+
+{% block bodyclass %}user ban-user{% endblock %}
+
+{% block title %}
+  {{ _('Ban') }}
+  {{ detail_user }}
+{% endblock %}
+{% set styles = ('dashboards', 'jquery-ui') %}
+{% set classes = 'compare' %}
+
+{% block site_css %}
+  {{ super() }}
+  {% stylesheet 'users' %}
+  {% stylesheet 'user-banned' %}
+{% endblock %}
+
+{% block content %}
+  <section id="content">
+    <h1>
+      <a href="{{ detail_user.get_absolute_url() }}">{{ detail_user }}</a>
+      {{ _('identified as a spammer.') }}
+    </h1>
+
+    <h2>
+      {{ _('The following revisions were reverted:') }}
+    </h2>
+    <div id="revisions-reverted">
```

Comment:
We may need a (English-only) note that says something like "No revisions were actually reverted. This feature will be added in phase 2".